config/webpack.config.prod.js

```diff
@@ -1,12 +1,14 @@
 'use strict'
 
 const webpack = require('webpack')
+const {target} = require('./webpack.vars')
+const devtool = target === 'mobile' ? false : '#cheap-module-source-map'
 
 module.exports = {
   output: {
     filename: 'app.[hash].min.js'
   },
-  devtool: '#cheap-module-source-map',
+  devtool: devtool,
```

Comment:
you can read more documentation [here](http://webpack.github.io/docs/configuration.html#devtool)